### PR TITLE
Using token to authenticate user

### PR DESCRIPTION
**Pull request by alandoni**

### CODE REVIEW

**Comment by talesprates** [comment]
please @kiosia  and @cristiadu review this so i can merge the fifabot code

**Review by talesprates — APPROVED** [review]
lgtm, i have tested with fifabot integration and worked as expected

File: api/controllers/adminController.js [hunk 1]
```diff
@@ -1,8 +1,7 @@
 'use strict'
-
+const sha3 = require('js-sha3').sha3_224;
```
Comment (cristiadu) [inline api/controllers/adminController.js:2]: can we not have written as this? the require just just include the require itself and then another variable to have the specific method (or whatever that is) that you want.

File: api/controllers/adminController.js [hunk 2]
```diff
@@ -52,13 +51,15 @@ class AdminController {
     _prepareToSend(adminSaved) {
         if (Array.isArray(adminSaved)) {
             let admins = util.copyObject(adminSaved);
-            for (const admin in admins) {
-                admins[admin].password = null;
+            for (const admin of admins) {
+                Reflect.deleteProperty(admin, 'password');
```
Comment (cristiadu) [inline api/controllers/adminController.js:55]: Correct me if I'm wrong, but this is the equivalent of a "Reflection"? If so, isn't this kind of a bad practice?

File: api/models/admin.js [hunk 3]
```diff
@@ -3,7 +3,8 @@ let Schema   = Mongoose.Schema;
 
 let adminSchema = new Schema({
     nickname : String,
-    password : String
+    password : String,
+    tokens : [String]
```
Comment (cristiadu) [inline api/models/admin.js:7]: does an user has multiple tokens? Why?

**Review by cristiadu — CHANGES_REQUESTED** [review]
See comments

File: api/controllers/adminController.js (hunk 2 above)
Comment (alandoni) [reply api/controllers/adminController.js:55]: this is it or "delete admin.password", which the lint says is not recomendable and suggests to do this way

File: api/models/admin.js (hunk 3 above)
Comment (alandoni) [reply api/models/admin.js:7]: because he can login on multiple devices

File: api/controllers/adminController.js (hunk 1 above)
Comment (alandoni) [reply api/controllers/adminController.js:2]: didn't understand what you wanted

Comment (cristiadu) [reply api/controllers/adminController.js:2]: const js_sha3 = require('js-sha3');

... (other code from imports)

const sha3 = js_sha3.sha3_224;

**Pull request merged**